.bazelci/presubmit.yml

```diff
@@ -34,7 +46,7 @@ platforms:
     test_targets:
     - "--"
     - "//tests/..."
-    - "-//tests/contrib:test_compare_ids_test_diff_ids_fails"
+    - "-//tests/contrib:test_compare_ids_test_*"
```

Comment:
Do we need the following lines after changing this pattern to a wildcard?

- "-//tests/contrib:test_compare_ids_test_invalid_tar_fails"
- "-//tests/contrib:test_compare_ids_test_invalid_tar_fails_multi_regex"
 - "-//tests/contrib:test_compare_ids_test_no_images_with_id_fails"

Reply:
ugh, this was a blanket replace error